src/BaGet/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -191,6 +193,15 @@ public static IServiceCollection AddBaGetContext(this IServiceCollection service
 
             return services;
         }
+        
+        public static IServiceCollection ConfigureIis(
+            this IServiceCollection services,
+            IConfiguration configuration)
+        {
+            services.ConfigureAndValidate<IISServerOptions>(configuration.GetSection(nameof(IISServerOptions)));
```

Comment:
I'd like to keep the appsettings.json file as light as possible. What do you think of setting the default of 250MB here? I think this should work:

```csharp
// IIS defaults upload sizes to ~30MB. Increase it to 250MB.
// See https://github.com/aspnet/Announcements/issues/267
services.Configure<IISServerOptions>(iis=>
{
    iis.MaxRequestBodySize = 262144000
});

services.ConfigureAndValidate<IISServerOptions>(configuration.GetSection(nameof(IISServerOptions)));
```

I believe this should default to 250MB, yet let customers override that value by adding a `IISServerOptions` option.